src/applications/personalization/view-dependents/components/ViewDependentsLayout.jsx

```diff
@@ -37,15 +38,11 @@ class ViewDependentsLayout extends Component {
         <div className="vads-l-grid-container">
           <div className="vads-l-row">
             <div className="vads-l-col--12 medium-screen:vads-l-col--8">
-              <div className="vads-l-row">
-                <div className="vads-l-col--12">
-                  <h1>Your Dependents</h1>
-                </div>
-              </div>
-              <ViewDependentsList />
+              <ViewDependentsHeader />
+              <ViewDepenentsList />
             </div>
             <div className="vads-l-col--12 medium-screen:vads-l-col--4">
-              <ViewDependentsSidebar />
+              <ViewDepenentsSidebar />
```

Comment:
This should be `ViewDependentsList` and `ViewDependentsSidebar`

Reply:
updated.